tests/regression/01-cpa/10-posneg.c

```diff
@@ -1,3 +1,4 @@
+//PARAM: --enable ana.int.wrap_on_signed_overflow
```

Comment:
Maybe add a comment on why this is needed here?

Reply:
I added a comment to the test case.